.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "mgclient"]
+	path = mgclient
+	url = git@github.com:memgraph/mgclient.git
```

Comment:
How do you ensure that the submodule is pulled?
Will we automate that or just document in the installation process the submodule init/update steps? 
This sounds okay to me if the user clones and tries to build pymgclient manually, but how does it work with PyPI and building from sources?

Reply:
When somebody want to build the package by cloning pymgclient, then they have to take care of pulling the submodule (I will document it in the README).

When somebdoy want to build the package by pulling from pypi, then the source will contain the mgclient library source also. Check the archive: https://test.pypi.org/project/pymgclient/#files . This way the source package become a bit larger, but eliminates git as dependency and the logic necessary to handle the submodule properly. The latter one is problematic, because obviously we don't want to ship the .git repo within the package, then we have to make sure git is working properly etc.. That's why I decided to package mgclient source into the source package.